Packs/Bitbucket/Integrations/Bitbucket/README.md

```diff
@@ -0,0 +1,3693 @@
+Pack for handling Bitbucket operations
+This integration was integrated and tested with version xx of Bitbucket
```

Comment:
Do we know the version here?
If yes, please fill it in, if not remove this line

Reply:
The version is 7.21.0. I updated it in the file